docs/index.asciidoc

```diff
@@ -0,0 +1,221 @@
+[[plugins-filters-dissect]]
```

Comment:
I was thinking about renaming when we copy the file, but on the other hand, that could break referencing between plugins doc.